packages/frontend/core/src/modules/cloud/services/auth.ts

```diff
@@ -228,4 +224,15 @@ export class AuthService extends Service {
 
     return headers;
   }
+
+  private setClientNonce() {
+    if (
+      BUILD_CONFIG.isElectron ||
+      BUILD_CONFIG.isIOS ||
+      BUILD_CONFIG.isAndroid
+    ) {
+      // send random client nonce on desktop app or mobile app
+      this.store.setClientNonce(nanoid());
```

Comment:
@forehalo Here we need to discuss whether to change the clientNonce each time, because if it's different each time, then when the user triggers the magic link login multiple times, only the last magic link received in the email will be valid, and opening the previous ones will be invalid.

Reply:
ban the previous otp is reasonable